crates/node/src/lib.rs

```diff
@@ -512,12 +515,15 @@ async fn handle_line(node: &mut Node, line: String) -> eyre::Result<()> {
                         println!("{IND} Left context {}", context_id);
                     }
                     "create" => {
-                        let Some((application_id, private_key)) = args.and_then(|args| {
-                            let mut iter = args.split(' ');
-                            let application = iter.next()?;
-                            let private_key = iter.next();
-                            Some((application, private_key))
-                        }) else {
+                        let Some((application_id, private_key, context_id)) =
+                            args.and_then(|args| {
+                                let mut iter = args.split(' ');
+                                let application = iter.next()?;
+                                let context_id = iter.next();
```

Comment:
I don't think we can reasonably support context ID here, since `private_key` is also optional

let's maybe forbid context specification from here, at least for now

we can beef it up later to provide some disambiguation between these entries

cause right now, if I don't want to specify a context id but do a private key

I'll have no choice but to specify both or my private key will be used as the context ID

and subscribe will announce it on the network, yay 😅

Reply:
I would then rather remove private key as we are not using it anyway

Reply:
Ah missed this, just refreshed